### DISCUSSION TITLE
What is left to do?

### DISCUSSION THREAD

**Discussion by asim** [micro/go-micro#2499] [General]
Go Micro has been around for 7 years now. We've separately created an org github.com/go-micro for the community ecosystem managed by @xpunch. I'm now wondering, is there anything left to do? It feels like the interfaces are stable and Go Micro operates mostly as interface definitions for high level distributed systems concerns. We may add new ones over time but with more oversight than in the past.  Otherwise, the best thing to do is strip out non relevant code and ensure robustness and stability moving forward.

Thoughts welcome.

**Comment by Davincible** [comment]
Better docs/walkthroughs could be valuable to this project I think, but I'm not sure if that's an answer you're looking for

**Reply by asim** [reply]
Then I welcome external contribution for those
